cogs/make_member.py

```diff
@@ -248,15 +250,15 @@ async def make_member(self, ctx: TeXBotApplicationContext, group_member_id: str)
             guest_role: discord.Role = await self.bot.guest_role
         except GuestRoleDoesNotExistError:
             logger.warning(
-                "\"/makemember\" command used but the \"Guest\" role does not exist. "
-                "Some user's may now have the \"Member\" role without the \"Guest\" role. "
-                "Use the \"/ensure-members-inducted\" command to fix this issue.",
+                '"/makemember" command used but the "Guest" role does not exist. '
+                'Some user\'s may now have the "Member" role without the "Guest" role. '
+                'Use the "/ensure-members-inducted" command to fix this issue.',
```

Comment:
Prefer strings to use double quotes & escape interior double quotes rather than using single quotes